libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/RustMatrixClient.kt

```diff
@@ -158,11 +168,18 @@ class RustMatrixClient(
             Timber.w("didRefreshTokens()")
             appCoroutineScope.launch {
                 val existingData = sessionStore.getSession(client.userId()) ?: return@launch
+                val anonymizedToken = client.session().accessToken.md5()
+                Timber.d("Saving new session data with token: $anonymizedToken. Is new token valid: ${existingData.isTokenValid}")
```

Comment:
I am confused about the `isTokenValid` set to `existingData.isTokenValid` at line 174 below. Should it be set to `true` in the case of a coming refresh token?

Also, the wording `Is new token valid` should be changed to `Was token valid`.

Reply:
Technically, you wrote that code so if you don't know, me neither 😅 . But yes, it's probably better to hardcode a `true` value there and fix the log.

Reply:
Yes, I know that I wrote the code, but it was a long time ago :).